build/ci/evergreen.yml

```diff
@@ -78,6 +78,78 @@ functions:
           E2E_TAGS: e2e,atlas
           <<: *go_options
         command: make e2e-test
+  "atlas logs e2e":
+    - command: subprocess.exec
+      type: test
+      params:
+        add_to_path:
+          - ${go_bin}
+        working_dir: src/github.com/mongodb/mongocli
+        env:
+          MCLI_OPS_MANAGER_URL: ${mcli_ops_manager_url}
+          MCLI_ORG_ID: ${atlas_org_id}
+          MCLI_PROJECT_ID: ${atlas_project_id}
+          MCLI_PRIVATE_API_KEY: ${atlas_private_api_key}
+          MCLI_PUBLIC_API_KEY: ${atlas_public_api_key}
+          MCLI_SERVICE: cloud
+          TEST_CMD: gotestsum --junitfile e2e-tests.xml --
+          E2E_TAGS: e2e,atlas,logs
+          <<: *go_options
+        command: make e2e-test
+  "atlas metrics e2e":
+    - command: subprocess.exec
+      type: test
+      params:
+        add_to_path:
+          - ${go_bin}
+        working_dir: src/github.com/mongodb/mongocli
+        env:
+          MCLI_OPS_MANAGER_URL: ${mcli_ops_manager_url}
+          MCLI_ORG_ID: ${atlas_org_id}
+          MCLI_PROJECT_ID: ${atlas_project_id}
+          MCLI_PRIVATE_API_KEY: ${atlas_private_api_key}
+          MCLI_PUBLIC_API_KEY: ${atlas_public_api_key}
+          MCLI_SERVICE: cloud
+          TEST_CMD: gotestsum --junitfile e2e-tests.xml --
+          E2E_TAGS: e2e,atlas,metrics
+          <<: *go_options
+        command: make e2e-test
+  "atlas clusters e2e":
+    - command: subprocess.exec
+      type: test
+      params:
+        add_to_path:
+          - ${go_bin}
+        working_dir: src/github.com/mongodb/mongocli
+        env:
+          MCLI_OPS_MANAGER_URL: ${mcli_ops_manager_url}
+          MCLI_ORG_ID: ${atlas_org_id}
+          MCLI_PROJECT_ID: ${atlas_project_id}
+          MCLI_PRIVATE_API_KEY: ${atlas_private_api_key}
+          MCLI_PUBLIC_API_KEY: ${atlas_public_api_key}
+          MCLI_SERVICE: cloud
+          TEST_CMD: gotestsum --junitfile e2e-tests.xml --
+          E2E_TAGS: e2e,atlas,clusters
+          <<: *go_options
+        command: make e2e-test
+  "iam e2e":
```

Comment:
looks like we are no longer testing this for cloud manager, I'm ok to split it out to a separate task but let's add one for Cloud manager as well